packages/order-utils/src/types.ts

```diff
@@ -25,6 +25,15 @@ export interface CreateOrderOpts {
     expirationTimeSeconds?: BigNumber;
 }
 
+export type SignatureValidationOpts = ValidatorSignatureOpts | PresignedSignatureOpts;
+
+export interface ValidatorSignatureOpts {
+    exchangeAddress: string;
+}
+
+export interface PresignedSignatureOpts {
+    exchangeAddress: string;
```

Comment:
What other opts are we imagining could be added? Maybe we should just pass the `exchangeAddress` directly as an opt?

Reply:
Yeah, felt a little unnecessary when I was writing it. Will just be passing the exchangeAddress to the `isValidSignatureAsync` then. Done in 7228cbfe92ff6fee4bc59681fba1789bec85c4f9.